<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCourse</class>
 <widget class="QDialog" name="addCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>651</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
   </widget>
   <widget class="QLabel" name="price">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="add_price">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Inter;
font-style: normal;
font-weight: bold;
font-size: 14px;
line-height: 32px;
color: #858585;</string>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="placeholderText">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_description">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Inter;
font-style: normal;
font-weight: bold;
font-size: 14px;
line-height: 32px;
color: #858585;</string>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="placeholderText">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="profile_edit_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: bold;
font-size: 30px;
line-height: 23px;

color: #949494;</string>
    </property>
    <property name="text">
     <string>Adicionar Curso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Inter</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Inter;
font-style: normal;
font-weight: bold;
font-size: 18px;
line-height: 32px;

text-align: center;
letter-spacing: 0.2px;

color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
